man1/aur-vercmp.1

```diff
@@ -4,52 +4,52 @@ aur\-vercmp \- check packages for AUR updates
 
 .SH SYNOPSIS
 .SY "aur vercmp"
-.OP \-acq
+.OP \-ac
 .OP \-p path
+.OP \-u filename
+
+.SH DESCRIPTION
+.B aur\-vercmp
+compares packages from
+.B stdin
+.RI ( pkgname
+and
+.IR pkgver ,
+space-separated) with their respective versions in the AUR, and prints
+any updated versions to
+.BR stdout .
 
 .SH OPTIONS
 .TP
-.B \-a
-If
-.B \-c
-is
-.I not
-specified, this option also shows packages with older or the same
-version in the AUR. If specified twice, packages that are not
-available in AUR are also shown.
+.BR \-a ", " \-\-all
+Show packages with an older or equal version in the AUR. (Not
+applicable to
+.BR \-c .)
 
 .TP
 .B \-c
-Takes packages and their versions from stdin
-.RI ( pkgname
-as the first field,
-.I pkgver
-as the second, space-separated), printing packages with equal or newer
-versions in a repository specified with
-.B \-d
-to stdout.
-
-If this argument is
-.I not
-specified, compare packages in the repository with their respective
-versions in the AUR, and print updates to stdout.
+Changes format to print packages with an equal or newer version to
+stdout.
```

Comment:
Maybe we should mention this detail:
```suggestion
.BR stdout .\&
A human\-readable version of the comparison will be printed to
.BR stderr .\&
E.g.: aurutils-git: 2.0.0 is newer than 1.5.3
```

While looking into this, I also found out that `plain` `msg1` and `msg2` will print to `stdout` instead of `stderr` [in the future].(https://git.archlinux.org/pacman.git/commit/scripts/libmakepkg/util/message.sh.in?id=3561c872ca0980c76442fcbf3985cc92a8f572f7)  We may want to protect ourselves from this. But lets leave it to its own PR.

Everything else looks good to me. 👍

Reply:
Yeah, I guess #473 might be related. I don't really want to have it print to stdout though (why?), so in a separate PR I might append >&2 to all msg/msg2.